docs/PROJECTS_API.md

```diff
@@ -0,0 +1,198 @@
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.updateProjectPartial(5000658, { customTitle: 'Graduation' })
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/projects/update-project-partial.js)
+
+
+### Delete a Specific Project
+
+Deletes a specific project.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.deleteProject(5000658)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+
+```
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/projects/delete-project.js)
+
+
+### Delete Specific Project Videos
+
+Deletes specific project videos. The quality parameter is optional.
+
+**IMPORTANT**: If you want to delete only a single quality video, you have to specify quality parameter, 
+otherwise all quality videos of the project will be deleted.
+
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.deleteProjectVideos(4120385, 360)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/projects/delete-project-videos.js)
+
+
+### Apply Template Preset on the Project
+
+Applies template preset on the project.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.applyTemplatePresetOnProject(5000658, 55)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/projects/apply-template-preset-on-project.js)
+
+
+### Duplicate the Project
+
+Duplicates the project.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.duplicateProject(5000658)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/projects/duplicate-project.js)
+
+
+### Render the Project
+
+Renders the project with given quality. The possible values for the quality are: 0, 360, 720, and 1080. 
+The watermark parameter is optional, must be in '.png' file format and have canvas size of 1920 x 1080 pixels,
+url length must not exceed 250 characters and is not applicable to HD quality videos.
```

Comment:
Remove this ` and is not applicable to HD quality videos.`
It is not actual anymore.

Reply:
will be fixed